### PR TITLE
No need to style sr-only elements

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/assets/stylesheets/blacklight/_constraints.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
   @extend .mb-2;
 }
 
-span.constraints-label {
+.constraints-label {
```
Comment (cbeer) [inline app/assets/stylesheets/blacklight/_constraints.scss]: Is this only used to style this `.sr-only` element?

https://github.com/projectblacklight/blacklight/blob/e7d7cb883e4d24a5263797cac47a8e5fd5b603ca/app/views/catalog/_constraints.html.erb#L4

Comment (jcoyne) [inline app/assets/stylesheets/blacklight/_constraints.scss]: Yes, as far as I can tell.

Comment (cbeer) [inline app/assets/stylesheets/blacklight/_constraints.scss]: Can we just remove the style entirely? No reason to make some sr-only text bold, is there?

Comment (jcoyne) [inline app/assets/stylesheets/blacklight/_constraints.scss]: Yes, that's best. We didn't have sr-only in the 6 branch, so I didn't notice.

**Comment by atz** [comment]
@jcoyne, @cbeer So we can close this or rework it?

**Pull request merged**